pkg/machine/qemu/options_darwin.go

```diff
@@ -2,12 +2,30 @@ package qemu
 
 import (
 	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
 )
 
+var podmanQemuDir = "/opt/homebrew/opt/podman/libexec/bin"
```

Comment:
Should this one match the directory below?
"/opt/homebrew/opt/podman/libexec/share/qemu"?